PowerEditor/installer/nsisInclude/mainSectionFuncs.nsh

```diff
@@ -81,6 +81,7 @@ Function copyCommonFiles
 	File "..\bin\langs.model.xml"
 	File "..\bin\stylers.model.xml"
 	File "..\bin\contextMenu.xml"
+	File "..\bin\toolbarIcons.xml"
```

Comment:
Should be `..\src\toolbarIcons.xml`

Reply:
And move up this line just behind `File "..\src\tabContextMenu_example.xml"` - due to `SetOverwrite on` (overwrite is turned ON, but installer should not overwrite user's data - icon could be already set).

Reply:
@donho 

But `toolbarIcons.xml` should be add only here (after this line):
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/f4517ef17e8dda08472166eda10d5cc3c52bcde4/PowerEditor/installer/nsisInclude/mainSectionFuncs.nsh#L77
where we have `SetOutPath "$UPDATE_PATH\"`

or also here (after this line)?
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/f4517ef17e8dda08472166eda10d5cc3c52bcde4/PowerEditor/installer/nsisInclude/mainSectionFuncs.nsh#L87
where we have `SetOutPath "$INSTDIR\"`